src/libraries/System.Drawing.Common/src/System/Drawing/Drawing2D/Matrix.cs

```diff
@@ -82,24 +82,18 @@ public Matrix Clone()
             return new Matrix(clonedMatrix);
         }
 
-        public float[] Elements
+        public unsafe float[] Elements
         {
             get
             {
-                IntPtr buf = Marshal.AllocHGlobal(6 * 8); // 6 elements x 8 bytes (float)
```

Comment:
Previously it was allocating 8 bytes per element, but a float is 4... why the mismatch, and is that relevant?

Reply:
Appears to be a simple mistake. The underlying implementation (`REAL` is `float`):

``` C++
    VOID GetMatrix(REAL* m) const
    {
        m[0] = M11;
        m[1] = M12;
        m[2] = M21;
        m[3] = M22;
        m[4] = Dx;
        m[5] = Dy;
    }
```

Reply:
It shouldn't matter. 

https://docs.microsoft.com/en-us/windows/win32/api/gdiplusmatrix/nf-gdiplusmatrix-matrix-getelements is documented as needing it to be `6 * sizeof(REAL)` where `REAL`  is `typedef float REAL;` in `gdiplustypes.h`.